README.md

```diff
@@ -12,10 +12,15 @@ Required:
 
 
 ```bash
-git clone https://github.com/OlympusDAO/olympusdao.git
-cd olympusdao
-yarn install
-npm run start
+$ git clone https://github.com/OlympusDAO/olympusdao.git
+$ cd olympusdao
+
+# set up your environment variables
+$ cp .env.example .env
+
+# fill in your own values in .env, then =>
```

Comment:
Can you add something about going to https://www.alchemy.com/ for the Alchemy values?

Reply:
added commentary in https://github.com/OlympusDAO/olympus-frontend/pull/415/commits/536f552bb114551f82ac711dd82a77355986246f